CK2Plus_expanded/common/landed_titles/000_CK2Plus_expanded_landed_titles.txt

```diff
@@ -23,6 +23,8 @@ k_neustria = {
 
 	capital = 112 # Ile de France
 
+	dignity = 200
```

Comment:
```suggestion
	dignity = 210
```
So that karl doesn't change titles when he takes austrasia

Reply:
Might want to raise it a bit more, since de jure Austrasia is a lot bigger than Neustria. Dignity is how many _more_ counties the title is considered to have.

Reply:
updated